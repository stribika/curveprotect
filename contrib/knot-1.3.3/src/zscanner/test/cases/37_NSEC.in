$ORIGIN	.
$TTL	1

; OK
@	NSEC	.					; The simplest variant - without bitmap
@	NSEC	\0320\ \\\"\.\@\*.tld.			; Special chars in domain name
@	NSEC	.	TYPE0				; Minimal type number
@	NSEC	.	TYPE65535			; Maximal type number
@	NSEC	.	TYPE0 A NS			; First bitmap window
@	NSEC	.	TYPE0 TYPE256 TYPE512 TYPE32768	; First, second, third and 128. bitmap window
@	TYPE47	\# 1 00					; TYPE + Hexadecimal rdata
@	TYPE47	.					; TYPE
@	nsec	.					; Type in lower-case

; KO
@	NSEC
@	NSEC						; Empty rdata
@	NSEC	\# 0					; Hex empty rdata
@	NSEC	\#					; Missing hex length
@	NSEC	.	TYPE65536			; Type number overflow
@	NSEC	.	X				; Unknown type
@	NSEC	\# 1 0000				; Too long rdata
@	NSEC	\# 2 00					; Bad rdata length
