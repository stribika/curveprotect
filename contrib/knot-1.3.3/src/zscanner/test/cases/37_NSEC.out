OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=082030205C222E402A03746C6400 
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 000180
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 FF200000000000000000000000000000000000000000000000000000000000000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 0001E0
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 000180010180020180800180
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=002F
RDATA=00 
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_BITMAP
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
